pekko-connectors-sample-http-csv-to-kafka/common/src/main/resources/logback.xml

```diff
@@ -31,8 +31,7 @@
 
     <logger name="org.apache" level="WARN"/>
     <logger name="kafka" level="WARN"/>
-    <logger name="akka" level="WARN"/>
-    <logger name="akka.kafka.benchmarks" level="INFO"/>
+    <logger name="org.apache.pekko" level="WARN"/>
     <logger name="org.apache.kafka.common.utils.AppInfoParser" level="ERROR"/>
```

Comment:
could you add back  `<logger name="org.apache.pekko.kafka.benchmarks" level="INFO"/>` ?